roles/devscripts/tasks/sub_tasks/_523_attach.yml

```diff
@@ -37,18 +37,44 @@
     vms: "{{ vms + [item] }}"
   loop: "{{ vms_list.stdout_lines }}"
 
+- name: "Power OFF virtual machine {{ item }}"
+  when:
+    - vms is defined
+    - vms | length > 0
+    - cifmw_devscripts_extra_networks | length > 1
+  community.libvirt.virt:
+    command: "destroy"
+    name: "{{ item }}"
+    uri: "qemu:///system"
+  loop: "{{ vms }}"
+
 - name: Attach an interface for each extra_network
   when:
     - vms is defined
     - vms | length > 0
   vars:
     vm_name: "{{ item[0] }}"
     network: "{{ item[1] }}"
+    offline: >-
+      {{
+        (cifmw_devscripts_extra_networks | length > 1) | ternary(true, false)
```

Comment:
"if true then true else false" :)
`| ternary()` is probably useless, at most `| bool` ?

Reply:
Thank you. You are right, I need `true` or `false`